wgengine/magicsock/magicsock.go

```diff
@@ -3038,7 +3038,7 @@ func (c *RebindingUDPConn) currentConn() nettype.PacketConn {
 // It returns the number of bytes copied and the source address.
 func (c *RebindingUDPConn) ReadFrom(b []byte) (int, net.Addr, error) {
 	for {
-		pconn := c.pconnAtomic.Load()
+		pconn := *c.pconnAtomic.Load()
```

Comment:
Is `c.pconnAtomic` gauranteed to be stored prior to this call? Otherwise, this could become a nil-pointer panic now.

Reply:
Based on my reading (where we call `rebind` in `NewConn`) it should always be stored. Though even in the old code, if there was no store there would have been a nil-pointer panic on the next line, when calling `pconn.ReadFrom`